roachtest: activerecord failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19075720?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19075720?buildTab=artifacts#/activerecord) on release-25.1 @ [e1f28a59cdabddf004bca68a45a3c54a08df3a4d](https://github.com/cockroachdb/cockroach/commits/e1f28a59cdabddf004bca68a45a3c54a08df3a4d):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.1-dev-e1f28a59cdabddf004bca68a45a3c54a08df3a4d
Tests run against activerecord 8.0.1
8559 Total Tests Run
8557 tests passed
2 tests failed
66 tests skipped
19 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_rename_enum_value (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-19075720/activerecord/1741757454702/1741759800194)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc target-release-25.2.0]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-48476

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19100059?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19100059?buildTab=artifacts#/activerecord) on release-25.1 @ [caa0fd66bd73ee373e10257cc6ffdac4f9308c25](https://github.com/cockroachdb/cockroach/commits/caa0fd66bd73ee373e10257cc6ffdac4f9308c25):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.2-dev-caa0fd66bd73ee373e10257cc6ffdac4f9308c25
Tests run against activerecord 8.0.1
8559 Total Tests Run
8557 tests passed
2 tests failed
66 tests skipped
19 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_rename_enum_value (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-19100059/activerecord/1741930454638/1741932719926)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142715 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.1.2-rc release-blocker]
- #142712 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc target-release-25.2.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19109452?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19109452?buildTab=artifacts#/activerecord) on release-25.1 @ [ac2b6541807c29116911dd08b96538935477667d](https://github.com/cockroachdb/cockroach/commits/ac2b6541807c29116911dd08b96538935477667d):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.2-dev-ac2b6541807c29116911dd08b96538935477667d
Tests run against activerecord 8.0.1
8559 Total Tests Run
8557 tests passed
2 tests failed
66 tests skipped
19 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_rename_enum_value (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-19109452/activerecord/1742016890306/1742019501932)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142715 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.1.2-rc release-blocker]
- #142712 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc target-release-25.2.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19141856?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19141856?buildTab=artifacts#/activerecord) on release-25.1 @ [830d73c903d4433ac4dccecab63f8cb18f90dea6](https://github.com/cockroachdb/cockroach/commits/830d73c903d4433ac4dccecab63f8cb18f90dea6):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.3-dev-830d73c903d4433ac4dccecab63f8cb18f90dea6
Tests run against activerecord 8.0.1
8559 Total Tests Run
8557 tests passed
2 tests failed
66 tests skipped
19 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_rename_enum_value (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/cpu_arch=arm64/run_1
```

Parameters:
 - <code>arch=arm64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-19141856/activerecord/1742362500557/1742365573811)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142715 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.2-rc]
- #142712 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-master]
- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc target-release-25.2.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -36,6 +36,7 @@ var activeRecordIgnoreList = blocklist{
 	`PostgresqlEnumTest#test_schema_dump_renamed_enum_with_to_option`:                                                                                          "affected by autocommit_before_ddl",
 	`PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum`:                                                                                        "affected by autocommit_before_ddl",
 	`PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum`:                                                                                          "affected by autocommit_before_ddl",
+	`PostgresqlInvertibleMigrationTest#test_migrate_revert_rename_enum_value`:                                                                                  "affected by autocommit_before_ddl",
 	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_not_save_and_return_false_if_a_callback_cancelled_saving_in_either_create_or_update`: "flaky",
 	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_not_update_children_when_parent_creation_with_no_reason`:                             "flaky",
 	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_update_children_when_autosave_is_true_and_parent_is_new_but_child_is_not`:            "flaky",

